system-contracts/bootloader/test_infra/src/main.rs

```diff
@@ -206,19 +203,15 @@ fn execute_internal_bootloader_test() {
         }
 
         match &test_result.unwrap() {
-            Ok(_) => println!("{} {}", "[PASS]".green(), test_name),
+            Ok(_) => {},
             Err(error_info) => {
                 tests_failed += 1;
-                println!("{} {} {}", "[FAIL]".red(), test_name, error_info)
```

Comment:
these are actual test logs that should be kept. 

you should keep the old println and just not add new ones unless needed

Reply:
Resolved.